stable/luigi/Chart.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+description: A Helm chart for Kubernetes
```

Comment:
Can you update the description to something about luigi. This description is used by tools like https://hub.kubeapps.com